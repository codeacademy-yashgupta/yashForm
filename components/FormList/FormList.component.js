import React, { Component } from 'react';
import { View } from 'react-native';
import FormCard from '../FormCard/FormCard.component';

export default class FormList extends Component {
  render() {
    const rows = [];
    for (let i = 0; i < 2; i += 1) {
      rows.push(<FormCard />);
    }
    return (
      <View>
        {rows}
      </View>
    );
  }
}
