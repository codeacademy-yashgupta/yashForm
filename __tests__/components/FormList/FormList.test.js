import React from 'react';
import renderer from 'react-test-renderer';
import FormList from '../../../components/FormList/FormList.component';
import 'react-native';

describe('FormList', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<FormList />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
