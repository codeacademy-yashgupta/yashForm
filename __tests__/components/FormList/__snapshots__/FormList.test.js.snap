// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormList should render correctly 1`] = `
<View>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "height": 200,
        "margin": 20,
        "padding": 20,
        "width": 300,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "black",
          "fontSize": 22,
          "fontWeight": "bold",
          "height": 100,
        }
      }
    >
      Hi
    </Text>
    <View
      style={
        Object {
          "borderBottomColor": "black",
          "borderBottomWidth": 1,
          "marginBottom": 20,
          "marginTop": 20,
        }
      }
    />
    <Text
      style={
        Object {
          "bottom": 0,
          "color": "black",
          "marginBottom": 10,
          "marginRight": 15,
          "position": "absolute",
          "right": 0,
        }
      }
    >
      Created On: 15/01/2019
    </Text>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "height": 200,
        "margin": 20,
        "padding": 20,
        "width": 300,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "black",
          "fontSize": 22,
          "fontWeight": "bold",
          "height": 100,
        }
      }
    >
      Hi
    </Text>
    <View
      style={
        Object {
          "borderBottomColor": "black",
          "borderBottomWidth": 1,
          "marginBottom": 20,
          "marginTop": 20,
        }
      }
    />
    <Text
      style={
        Object {
          "bottom": 0,
          "color": "black",
          "marginBottom": 10,
          "marginRight": 15,
          "position": "absolute",
          "right": 0,
        }
      }
    >
      Created On: 15/01/2019
    </Text>
  </View>
</View>
`;
